### PR TITLE
Reference official config docs on consul website

### PR DESCRIPTION
**Pull request by findkim**
A few outstanding dependencies before the links work:
* [x] Website has been deployed with NIA docs
* [ ] New tag has been created for Sync

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -4,6 +4,9 @@
 
 Consul Terraform Sync (just Sync from here on) is a service-oriented tool for managing network infrastructure near real-time. Sync runs as a daemon and integrates the network topology maintained by your Consul cluster with your network infrastructure to dynamically secure and connect services.
 
+* Website: https://www.consul.io/docs/nia
+* Forum: [Discuss](https://discuss.hashicorp.com/tags/c/consul/29/consul-terraform-sync)
```
Comment (eikenb) [inline README.md:8]: Going to this URL gives me this.... "Oops! That page doesn’t exist or is private."

Comment (eikenb) [reply README.md:8]: The Discuss link.

Comment (findkim) [reply README.md:8]: yeah the tag needs to be created

Comment (findkim) [reply README.md:8]: tag won't be created for a bit, removing this to be merged

**Pull request merged**